{{cookiecutter.project_slug}}/.pre-commit-config.yaml

```diff
@@ -65,3 +65,10 @@ repos:
         - commit
       types:
         - python
+  {%+ if cookiecutter.code_security == 'bandit' +%}
+  - repo: https://github.com/PyCQA/bandit
```

Comment:
bandit is already installed via poetry, so you don't need to install it again, inside the pre-commit
check the other options like mypy and replicate the same idea.